### PR TITLE
Setup GitHub Actions

### PR DESCRIPTION
**Pull request by ChristianTovar**
# Description

Just the regular yaml file for using GitHub Actions for this current Elixir app. Nothing fancy yet, just nice for having it from the beginning. If we end up having a monorepo we will adjust this file to fit our future needs.

### CODE REVIEW

File: .github/workflows/elixir.yml [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+
+    name: Build and test
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Elixir
+      uses: erlef/setup-beam@988e02bfe678367a02564f65ca2e37726dc0268f
+      with:
+        elixir-version: '1.12.3'
```
Comment (ZachLiss) [inline .github/workflows/elixir.yml]: In our `mix.exs` we're using elixir 1.9, and the latest is 1.13.3. What do you think of upgrading this?

File: lib/boat_tracker/application.ex [hunk 2]
```diff
@@ -13,7 +13,7 @@ defmodule BoatTracker.Application do
     opts = [strategy: :one_for_one, name: BoatTracker.Supervisor]
 
     # Children for all targets
-    children = [GPS | children(target())]
+    children = children(target())
```
Comment (ZachLiss) [inline lib/boat_tracker/application.ex]: I think we can discard this change since it's already in the Parser PR

Comment (ChristianTovar) [reply lib/boat_tracker/application.ex]: Tests will fail if this change is not present. This change does not `init` the `UART` for hosts (everything except the RPi).

File: .github/workflows/elixir.yml (hunk 1 above)
Comment (ChristianTovar) [reply .github/workflows/elixir.yml]: Yeah, sounds perfect. But do you mean on both the `mix.exs` and yaml file? or just yaml file?

Comment (ZachLiss) [reply .github/workflows/elixir.yml]: Let's do the yaml file here, then we can have a different PR for the `mix.exs`

**Pull request merged**